storage/indexer.go

```diff
@@ -160,6 +160,7 @@ func (i *Indexer) updateIndex(ctx context.Context) error {
 	defer i.m.Unlock()
 	i.cursor = storageCursor.Current
```

Comment:
Could we also save the `cursor.Current`? It's a string value. It would be great to observe how it changes.

Reply:
If it is a string, not sure how to do that because it's not a counter neither a gauge.
What is the format of that cursor? Could it be casted to float64 that string ? If so, I could set a gauge metric and set that value on each update.

Reply:
No, it might be dangerous if we forget about its format :) In the worst case, we can collect "cursor" as logs.